### ISSUE TITLE
The adapter @astrojs/deno doesn't provide a feature map. It is required in Astro 4.0

### ISSUE THREAD

**Issue by Mac-Mann** [denoland/deno-astro-adapter#13]
As mentioned in the title, I keep seeing this error when attempting to run Astro's build step. I see that recently the Deno team recently updated the repo here to utilize Astro 4.0, but it seems they might have missed a step. 😅

**Comment by skrzepij** [comment]
@Mac-Mann yeah, there are some issues with updating the package in npm registry as astro team remains to be the owner of `@astrojs/deno` package 😄 
for now you can update your package.json to install adapter directly from this repository, works for me.

`"@astrojs/deno": "github:denoland/deno-astro-adapter",`

**Comment by Mac-Mann** [comment]
@skrzepij Thanks so much for sharing this! That seemed to do the trick, much appreciated! 👍🏻
